src/_guides/language/language-tour.md

```diff
@@ -2989,6 +2989,12 @@ Vector class, you might define a `+` method to add two vectors.
 `–`  | `%`  | `>>`
 {:.table}
 
+<div class="alert alert-info" markdown="1">
+  **Note:** You may have noticed that `!=` is not an overridable operator.
+  This is because `!=` is not an operator in its own right; the expression `e1 != e2`
+  is just syntactic sugar for `!(e1 == e2)`.
```

Comment:
Great note! I'd suggest that we avoid stating that `!=` isn't "an operator in its own right". Maybe simplify this sentence to the following? @kwalrath, WDYT?

> This is because `!=` is just syntactic sugar for `!(e1 == e2)`.

Reply:
I was thinking about something that, but I didn't want to have too much back and forth. I'll make this change later.